### PR TITLE
feat: add sideMenu navigation and style

### LINKED ISSUE #12: Add side menu

### PR DESCRIPTION
**Pull request by willtrev**
Add navigation and style to side menu
closes #12

### CODE REVIEW

File: src/components/SideMenu/Tab.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import './Tab.styles.scss';
+
+interface TabProps extends HTMLElement {
```
Comment (lucas-giraldelli) [inline src/components/SideMenu/Tab.tsx]: `HTMLAnchorElement` then don't need href prop to be declared

File: src/components/SideMenu/Tab.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import './Tab.styles.scss';
```
Comment (lucas-giraldelli) [inline src/components/SideMenu/Tab.tsx]: Also tab could be called `SideMenuTab` or `MenuTab`

File: src/components/SideMenu/SideMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import './SideMenu.styles.scss';
+import ChurchIcon from '@mui/icons-material/Church';
+import SettingsIcon from '@mui/icons-material/Settings';
+import LogoutIcon from '@mui/icons-material/Logout';
+import Tab from './Tab';
```
Comment (lucas-giraldelli) [inline src/components/SideMenu/SideMenu.tsx]: I think we should configure `eslint/sort-import` or something like, to make npm imports first, then components (tsx) then css/utils last.

File: src/components/SideMenu/Tab.tsx (hunk 1 above)
Comment (willtrev) [reply src/components/SideMenu/Tab.tsx]: done

File: src/components/SideMenu/Tab.tsx (hunk 2 above)
Comment (willtrev) [reply src/components/SideMenu/Tab.tsx]: done

File: src/components/SideMenu/SideMenu.tsx (hunk 3 above)
Comment (willtrev) [reply src/components/SideMenu/SideMenu.tsx]: done

**Pull request merged**